src/wiki/plugins/links/mdx/djangowikilinks.py

```diff
@@ -118,8 +118,12 @@ def handleMatch(self, m):
                 path = self.config['base_url'][0] + path_from_link
 
         label = m.group('linkTitle')
+        fragment = m.group('fragment')
+        if not fragment:
+            fragment = ""
+
         a = etree.Element('a')
-        a.set('href', path)
+        a.set('href', path+fragment)
```

Comment:
Because of some new PEP8 rules, this will break - but will fix it up after + write a release note.